src/sections/wallet/assets/farmingPositions/WalletFarmingPositions.utils.tsx

```diff
@@ -41,65 +42,76 @@ export const useFarmingPositionsTable = (data: FarmingPositionsTableData[]) => {
     position: true,
   }
 
-  const columns = [
-    accessor("symbol", {
-      id: "name",
-      header: isDesktop
-        ? t("wallet.assets.farmingPositions.header.name")
-        : t("selectAssets.asset"),
-      sortingFn: (a, b) => a.original.symbol.localeCompare(b.original.symbol),
-      cell: ({ row }) => <WalletAssetsTableName {...row.original} />,
-    }),
-    accessor("date", {
-      id: "date",
-      header: t("wallet.assets.farmingPositions.header.date"),
-      sortingFn: (a, b) => (isAfter(a.original.date, b.original.date) ? 1 : -1),
-      cell: ({ row }) => (
-        <Text fs={16} fw={500} color="white">
-          {t("wallet.assets.farmingPositions.data.date", {
-            date: row.original.date,
-          })}
-        </Text>
-      ),
-    }),
-    accessor("shares", {
-      id: "shares",
-      header: t("wallet.assets.farmingPositions.header.shares"),
-      sortingFn: (a, b) => (a.original.shares.gt(b.original.shares) ? 1 : -1),
-      cell: ({ row }) => (
-        <Text fs={16} fw={500} color="white">
-          {t("value.token", { value: row.original.shares })}
-        </Text>
-      ),
-    }),
-    accessor("position", {
-      id: "value",
-      header: t("wallet.assets.farmingPositions.header.value"),
-      sortingFn: (a, b) =>
-        a.original.position.valueDisplay.gt(b.original.position.valueDisplay)
-          ? 1
-          : -1,
-      cell: ({ row }) => (
-        <div sx={{ flex: "column", align: ["end", "start"], gap: 2 }}>
-          <WalletAssetsHydraPositionsData
-            symbol={row.original.position.symbol}
-            value={row.original.position.value}
-            lrna={row.original.position.lrna}
-          />
-          <DollarAssetValue
-            value={row.original.position.valueDisplay}
-            wrapper={(children) => (
-              <Text fs={[11, 12]} lh={[14, 16]} color="whiteish500">
-                {children}
-              </Text>
-            )}
-          >
-            <DisplayValue value={row.original.position.valueDisplay} />
-          </DollarAssetValue>
-        </div>
-      ),
-    }),
-  ]
+  const columns = useMemo(
```

Comment:
Why did you add the useMemo hook with no dependencies?

Reply:
It's not necessary to recreate column array for each table render. There are no external dependencies needed to create these columns and that's why dependency array is empty.

Reply:
There is a note in tanstack doc: 

> Note: If defining columns inside a component, you should still try to give the column definitions a stable identity. This will help with performance and prevent unnecessary re-renders. Store the column definitions in either a useMemo or useState hook.

Reply:
okay, nice then. Just fix a lint issue caused by the no dependencies

Reply:
@vkulinich-cl  Looks like there're no eslint erros even after rebase.

Reply:
React Hook useMemo has missing dependencies: 'accessor', 'isDesktop', and 't'. Either include them or remove the dependency array. I have this lint warning caused by empty hook dependencies

Reply:
Got it, thanks 👍